ruby/main.rb

```diff
@@ -12,6 +12,8 @@
 # This must be imported after sentry-ruby for transactions to work
 require "sinatra"
 require "sinatra/cors"
+require "google-cloud-datastore"
+require "sinatra/activerecord"
```

Comment:
@bfpepper87 let's get rid of this if it's not being used. Please make sure the Gemfile didn't get updated with this too